From 906b38e039a4f1140092bafdab65e5aec18ecdb4 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Fri, 29 Jul 2022 09:38:53 +0300
Subject: [PATCH] [SW-95911] gaudi2: pll: add pll.c in ppboot build

This patch adds gaudi2 specific pll.c file for PPBOOT
build. PLL functions are required to configure SRAM PLLs
before running SRAM BIST.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: Ic4b3c21110e1b9a08fa5fb6e90b94e25b0141cdf
---

diff --git a/arch/gaudi2/common/Makefile b/arch/gaudi2/common/Makefile
index 0450740..e0291d2 100644
--- a/arch/gaudi2/common/Makefile
+++ b/arch/gaudi2/common/Makefile
@@ -9,6 +9,7 @@
 
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/common.c)
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/$(HL_TARGET).c)
+PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/pll.c)
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/security.c)
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/tpm2.c)
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/tpm_auth.c)
